Quickstart.md

```diff
@@ -0,0 +1,197 @@
+# QuickStart guide on how to setup AppCli with a sample application
+
+### Prerequisites
+
+- python
+- docker
+- docker-compose
+
+
+### Setup the appcli enironment
+
+#### Create the required directory structure
+
+Run the following commands to create the required files and folders
+
+```
+mkdir -p src/resources/templates/{baseline,configurable}
+touch src/resources/myapp.py
+touch src/resources/settings.py
+touch src/resources/stack-settings.py
+```
+
```

Comment:
Unnecessary empty line, just add one. The markdown formatter will ignore the extra whitespace, so its useless putting it in.

Clean up through this file